app/controllers/persistent_volume_controller.rb

```diff
@@ -1,15 +1,13 @@
 class PersistentVolumeController < ApplicationController
-  include ContainersCommonMixin
   before_action :check_privileges
   before_action :get_session_data
   after_action :cleanup_action
   after_action :set_session_data
 
-  def show_list
-    process_show_list
-  end
+  include ContainersCommonMixin
+  include GenericListMixin
```

Comment:
..ContainersCommonMixin has that `included do...`, is GenericListMixin still needed here?

And, if so, why not `Mixin::GenericListMixin`?